mona_sdk/client.py

```diff
@@ -103,10 +103,12 @@
 
 # When set, SDK will randomly sample the sent data for any class keyed in the config.
 # See readme for more details.
-SAMPLING_CONFIGURATION_DICT = get_dict_value_for_env_var(
-    "MONA_SDK_SAMPLING_CONFIG", cast_values=float
+SAMPLING_CONFIGURATION_DICT = (
+    get_dict_value_for_env_var("MONA_SDK_SAMPLING_CONFIG", cast_values=float) or {}
```

Comment:
get_dict_value_for_env_var accepts default_value param, please use it instead of the or